src/meltano/core/plugin/singer/tap.py

```diff
@@ -185,6 +185,9 @@ class SingerTap(SingerPlugin):  # noqa: WPS214
             value_processor="nest_object",
         ),
         SettingDefinition(name="_select_filter", kind=SettingKind.ARRAY, value=[]),
+        SettingDefinition(
+            name="_no_catalog_cache", kind=SettingKind.BOOLEAN, value=False
```

Comment:
Nit that came up while checking this out during Office Hours:

What do you think of making this `_catalog_cache` instead and default it to `True`?

Reply:
All else being equal, avoiding a double negative would be nice